autogen/main.tf.tmpl

```diff
@@ -64,25 +66,27 @@ resource "google_compute_global_address" "default" {
 
 ### IPv6 block ###
 resource "google_compute_global_forwarding_rule" "http_ipv6" {
-  provider   = google-beta
-  project    = var.project
-  count      = (var.enable_ipv6 && local.create_http_forward) ? 1 : 0
-  name       = "${var.name}-ipv6-http"
-  target     = google_compute_target_http_proxy.default[0].self_link
-  ip_address = local.ipv6_address
-  port_range = "80"
-  labels     = var.labels
+  provider              = google-beta
+  project               = var.project
+  count                 = (var.enable_ipv6 && local.create_http_forward) ? 1 : 0
+  name                  = "${var.name}-ipv6-http"
+  target                = google_compute_target_http_proxy.default[0].self_link
+  ip_address            = local.ipv6_address
+  port_range            = "80"
+  labels                = var.labels
+  load_balancing_scheme = var.load_balancing_scheme
 }
 
 resource "google_compute_global_forwarding_rule" "https_ipv6" {
-  provider   = google-beta
-  project    = var.project
-  count      = (var.enable_ipv6 && var.ssl) ? 1 : 0
-  name       = "${var.name}-ipv6-https"
-  target     = google_compute_target_https_proxy.default[0].self_link
-  ip_address = local.ipv6_address
-  port_range = "443"
-  labels     = var.labels
+  provider              = google-beta
+  project               = var.project
+  count                 = (var.enable_ipv6 && var.ssl) ? 1 : 0
+  name                  = "${var.name}-ipv6-https"
+  target                = google_compute_target_https_proxy.default[0].self_link
+  ip_address            = local.ipv6_address
+  port_range            = "443"
+  labels                = var.labels
+  load_balancing_scheme = var.load_balancing_scheme
```

Comment:
The only change here is the addition of the load_balancing_scheme and re-aligning the other field/value pairs